### PR TITLE
Run ww_activate script on linux workspace activation

### PR DESCRIPTION
**Pull request by lvoliveira**
On Linux, this script was already being created for this purpose, but it was not being executed. On Windows it was working correctly.

I have also added a test for it and improved tests a bit. Currently only run for linux.

### CODE REVIEW

**Review by nicoddemus — APPROVED** [review]
Nice!

**Review by lincrosenbach — APPROVED** [review]
💪🏼

File: testing/test_tools [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    rm -rf temp-ww-workdir
+    mkdir $test_dir/temp-ww-workdir
+    export WW_DEFAULT_PATH=$test_dir/temp-ww-workdir
+}
+
+tier_down() {
+    cd $test_dir
+    rm -rf temp-ww-workdir
+}
```
Comment (prusse-martin) [inline testing/test_tools:41]: I believe this is a typo and should be replaced by `teardown` (no?).

Comment (lvoliveira) [reply testing/test_tools]: Fixed 😅

**Review by prusse-martin — APPROVED** [review]

**Pull request merged**